src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -50,9 +50,17 @@ interface FilesTableRowProps {
   baseFieldName: string;
   showFileVersion: boolean;
   disabled: boolean;
+  archived?: boolean;
 }
 
-export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion, disabled }: FilesTableRowProps) => {
+export const FilesTableRow = ({
+  file,
+  removeFile,
+  baseFieldName,
+  showFileVersion,
+  disabled,
+  archived = false,
```

Comment:
Er ikke denne propen overflødig, da vi allerede har data for å vite dette i `file`?
```suggestion
```
Virker som `const archived = file.type === 'UnpublishableFile';` oppnår samme bare at vi baserer oss på den aktuelle fila i stedet for en ekstra prop

Reply:
Mulig man kan dra det enda lenger og fjerne `file`-propen også 🤔 
`const disabled = canEditFile(file);`
På den måten bestemmer komponenten litt mer selv basert på fila, i stedet for å motta både fila _og_ props som gjenforteller hvilken fil det er.

Dette er uansett ikke implementert i denne PRen da, så kan vurdere selv hva du vil gjøre her

Reply:
Godt poeng! Tar bort archived.

Men skjønte ikke helt hva du mente med å ta bort fila også?

Reply:
Ups, det var en skrivefeil ja 😅 Mente `disabled`-propen

Reply:
Ja, jeg tror det er en god idé men endrer det når jeg rekatorerer Fila og lager flere komponenter, for den er allerede veldig stor så har ikke så lyst til å flytte inn mer kode enda.